functions/convert/main.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+
+# Method for converting MD files to HTML
+
+from __future__ import print_function, unicode_literals
+
+import codecs
+import os
+import re
+import shutil
+import tempfile
+from contextlib import closing
+import markdown
+import boto3
+import datetime
+
+from glob import glob
+from general_tools.file_utils import unzip, add_file_to_zip, make_dir, write_file
+from general_tools.print_utils import print_error, print_ok
+from general_tools.url_utils import download_file
+
+event = {}
+context = {}
```

Comment:
@ethantkoenig Great points. I had done this in other lambda functions I ran just so everything has access to the request, both the event variable (which has the data key) and the context variable which holds info on the process. We could get rid of these, yet I always like to have them available if needed in further development. I am sure you have learned different/newer programming theory than us old(er) guys...but do you think it poses a problem? Is it best to just pass everything needed as arguments to a function?